apps/admin-portal/src/components/applications/application-list.tsx

```diff
@@ -16,110 +16,83 @@
-            value: "Usage",
-        },
-    ];
 
     return (
-        <>
-            { appList ? (
-                    <Grid>
-                        <Grid.Row divided>
-                            <Grid.Column width={ 3 }>
-                            <span>Sort By <Divider vertical hidden/>
-                            <Dropdown
-                                inline
-                                options={ sort }
-                                defaultValue={ sort[0].value }
-                            />
-                            </span>
-                            </Grid.Column>
-                            <Grid.Column width={ 3 }>
-                                <span> Total Applications { appList.totalResults } </span>
-                            </Grid.Column>
-                        </Grid.Row>
-                        <Grid.Row>
-                            <Grid.Column width={ 7 }/>
-                            <Grid.Column width={ 6 }>
-                                <Input icon="search" placeholder="Search Application ..."/>
-                            </Grid.Column>
-                            <Grid.Column width={ 3 } className="last-column">
-                                <Button onClick={ navigate }> +APPLICATION </Button>
-                            </Grid.Column>
-                        </Grid.Row>
-                        <Grid.Row>
-                            <Grid.Column>
-                                <Divider hidden/>
-                                <List divided verticalAlign="middle" className="application-list">
-                                    {
-                                        appList.applications.map((apps) => (
-                                            <ApplicationListItem
-                                                key={ apps.id }
-                                                name={ apps.name }
-                                                id={ apps.id }
-                                                accessUrl={ apps.accessUrl }
-                                                imageUrl={ apps.imageUrl }
-                                                description={ apps.description }
-                                                self={ apps.self }
-                                            />
-                                        ))
-                                    }
-                                </List>
-                            </Grid.Column>
-                        </Grid.Row>
-                    </Grid>
-                )
-                :
-                (
-                    <Grid>
-                        <Grid.Row divided>
-                            <Grid.Column width={ 10 }>
-                                <p>No application found</p>
-                            </Grid.Column>
-                        </Grid.Row>
-                    </Grid>
-                )
+        <AdminList className="applications-list">
```

Comment:
Can we rethink the component name 'AdminList' here?

Reply:
Any suggestions?